javalib/src/main/scala/java/util/logging/ConsoleHandler.scala

```diff
@@ -0,0 +1,19 @@
+package java.util.logging
+
+// Unfortunately the spec require the use of System.err
+class ConsoleHandler extends StreamHandler(System.err, new SimpleFormatter) {
```

Comment:
There is no `LogManager` in this port. I take that as that the properties are not defined so we use the default values. That's why I'm defaulting to `new SimpleFormatter`. It is not clear to me what's are you pointing wrong in your comment